quantum/nodes/classical-register/classical-register.html

```diff
@@ -117,13 +117,19 @@
     <p id="error-message" style="color: red"></p>
   </div>
   <div class="form-row">
-    <label for="node-input-number-bits"><i class="fa fa-tag"></i> Number of bits</label>
+    <label for="node-input-number-bits"
+      ><i class="fa fa-tag"></i> Number of bits</label
+    >
     <input type="number" min="1" id="node-input-classicalBits" />
   </div>
 </script>
 
 <script type="text/html" data-help-name="classical-register">
   <p>
-    Classical register node
+    This node is used for implementing a classical register in the quantum
```

Comment:
Not quite accurate I think:

The classical register is basically a family/group of classical bits under the same label, which will be used to measure & store the value of qubits. However, the only purpose is to label the classical bits rather than having a simple list. I believe in your paragraph, it can be understood that the classical register is required to measure qubits.